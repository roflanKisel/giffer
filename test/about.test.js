import React from 'react';
import { shallow } from 'enzyme';
import About from '../pages/about';

describe('About page', () => {
  it('should renders correctly', () => {
    const wrapper = shallow(<About />);

    expect(wrapper).toMatchSnapshot();
  });
});
