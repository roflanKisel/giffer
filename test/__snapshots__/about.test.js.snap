// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page should renders correctly 1`] = `
<div>
  <Navbar />
  <About />
</div>
`;
